@codex please fix

```
> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
 ✓ tests/eslintPluginPresence.test.ts (1 test) 6ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 5ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 4ms
 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 12ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 5ms
 ✓ tests/prismSvelte.test.ts (1 test) 13ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 4ms
 ✓ tests/progressionBalance.test.ts (1 test) 28ms
 ✓ tests/domException.test.ts (1 test) 5ms
 ✓ tests/indexeddb.server.test.ts (1 test) 289ms
 ✓ frontend/tests/sanity.test.ts (1 test) 4ms
[warn] src/pages/quests/svelte/Quest.svelte
[warn] Code style issues found in the above file. Run Prettier with --write to fix.
 ❯ scripts/tests/prettierFormatting.test.ts (1 test | 1 failed) 19562ms
   × Prettier formatting > frontend sources are formatted 19560ms
     → Command failed: npm run format:check

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  scripts/tests/prettierFormatting.test.ts > Prettier formatting > frontend sources are formatted
Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5
      6|   test('frontend sources are formatted', () => {
      7|     const frontendDir = path.join(__dirname, '../../frontend');
      8|     execSync('npm run format:check', {
       |     ^
      9|       cwd: frontendDir,
     10|       stdio: 'inherit',

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


 Test Files  1 failed | 290 passed (291)
      Tests  1 failed | 1034 passed (1035)
   Start at  23:23:03
   Duration  167.17s (transform 9.71s, setup 11.29s, collect 46.41s, tests 110.22s, environment 189.31s, prepare 35.42s)


Error: Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Serialized Error: { status: 1, signal: null, output: [ null, null, null ], pid: 9336, stdout: null, stderr: null }

 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/src/pages/quests/svelte/Quest.svelte
+++ b/frontend/src/pages/quests/svelte/Quest.svelte
@@ -5,23 +5,22 @@
 
     let imageLoaded = false;
 
-    $: statusLabel =
-        status === 'completed'
-            ? 'Completed'
-            : status === 'available'
-              ? ''
-              : status === 'locked'
-                ? 'Locked'
-                : 'Checking';
+    function getStatusLabel(currentStatus) {
+        if (currentStatus === 'completed') return 'Completed';
+        if (currentStatus === 'available') return '';
+        if (currentStatus === 'locked') return 'Locked';
+        return 'Checking';
+    }
 
-    $: assistiveStatusLabel =
-        status === 'available'
-            ? 'Available'
-            : status === 'completed'
-              ? 'Completed'
-              : status === 'locked'
-                ? 'Locked'
-                : statusLabel;
+    function getAssistiveStatusLabel(currentStatus, currentStatusLabel) {
+        if (currentStatus === 'available') return 'Available';
+        if (currentStatus === 'completed') return 'Completed';
+        if (currentStatus === 'locked') return 'Locked';
+        return currentStatusLabel;
+    }
+
+    $: statusLabel = getStatusLabel(status);
+    $: assistiveStatusLabel = getAssistiveStatusLabel(status, statusLabel);
 </script>
 
 <div class="container" class:quest data-testid="quest-tile" data-status={status}>

--- a/frontend/src/pages/quests/svelte/Quest.svelte
+++ b/frontend/src/pages/quests/svelte/Quest.svelte
@@ -9,8 +9,8 @@
         status === 'available' || status === 'completed'
             ? ''
             : status === 'locked'
-                ? 'Locked'
-                : 'Checking';
+              ? 'Locked'
+              : 'Checking';
 
     $: assistiveStatusLabel =
         status === 'available'

